www/scss/theme.scss

```diff
@@ -1,29 +1,38 @@
-@import url('https://fonts.googleapis.com/css?family=Inconsolata');
-
 $view-width: 60vw;
 
+$font-family: 'Inconsolata', -apple-system, BlinkMacSystemFont, 'Segoe UI', Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol';
```

Comment:
I'm pretty sure that this doesn't have to be that verbose, but I don't have the browsers/systems to test it.